microcluster/app.go

```diff
@@ -480,3 +480,27 @@ func (m *MicroCluster) SQL(ctx context.Context, query string) (string, *internal
 
 	return "", batch, err
 }
+
+// APIExtensions returns the list of API extensions registered with the daemon.
```

Comment:
"registered with every member of the cluster."?

Reply:
fixed